frontend/vite.config.ts

```diff
@@ -0,0 +1,37 @@
+import { defineConfig } from "vite";
+import vue from "@vitejs/plugin-vue";
+
+const SERVER_PORT = process.env.PORT || 3000;
+const HTTPS_PORT = 443;
+
+export default defineConfig(() => {
+  // NOTE: the following lines is to solve https://github.com/gitpod-io/gitpod/issues/6719
+  // tl;dr : the HMR(hot module replacement) will behave differently when VPN is on, and by manually set its prot to 443 should prevent this issue.
```

Comment:
nit: prot -> port

Reply:
My bad, I forgot to fix the typo when I copy &  paste the comment.

Add a commit https://github.com/bytebase/bytebase/pull/75/commits/b8806db95ba8e88c22453cd3dca0cc53e2b65c07.